database-commons/src/main/java/co/cask/db/batch/sink/AbstractDBSink.java

```diff
@@ -336,6 +338,16 @@ private void emitLineage(BatchSinkContext context, List<Schema.Field> fields) {
     @Description("Name of the database table to write to.")
     @Macro
     public String tableName;
+
+    /**
+     * Adds escape characters (back quotes, double quotes, etc.) to the table name for
```

Comment:
so the escape character used depends on the underlying DB?

Reply:
Also why does only the sink need this and not the source?

Reply:
escape character in PostgreSQL is impacting only the case sensitivity of column names, that's why we need to handle it in sink, but in source Users provides query with column names and we not infering the schema

Reply:
sg